plugin/action/throttle/throttle.go

```diff
@@ -170,7 +236,11 @@ func (p *Plugin) isAllowed(event *pipeline.Event) bool {
 			limiter, has = limiters[p.pipeline][limiterKey]
 			// we could already write it between `limitersMu.RUnlock()` and `limitersMu.Lock()`, so we need to check again
 			if !has {
-				limiter = NewLimiter(p.config.BucketInterval_, p.config.BucketsCount, rule.limit)
+				limiter = p.getNewLimiter(
+					string(p.config.ThrottleField),
+					pipeline.ByteToStringUnsafe(p.limiterBuff[2:]),
```

Comment:
Why [2:]?

Reply:
Higher in code we writes index of rule and `:` to buff:
```
		p.limiterBuff = append(p.limiterBuff[:0], byte('a'+index))
		p.limiterBuff = append(p.limiterBuff, ':')
		p.limiterBuff = append(p.limiterBuff, throttleKey...)
		limiterKey := pipeline.ByteToStringUnsafe(p.limiterBuff)
``

Reply:
This part was rewritten